lib/jobs/validate-ssh.js

```diff
@@ -0,0 +1,108 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var di = require('di');
+module.exports = validationJobFactory;
+
+di.annotate(validationJobFactory, new di.Provide('Job.Ssh.Validation'));
+di.annotate(validationJobFactory, new di.Inject(
+    'Job.Base',
+    'Util',
+    'Logger',
+    'Assert',
+    'Promise',
+    'Services.Waterline',
+    'ssh',
+    '_'
+));
+
+function validationJobFactory(
+    BaseJob,
+    util,
+    Logger,
+    assert,
+    Promise,
+    waterline,
+    ssh,
+    _
+) {
+    var logger = Logger.initialize(validationJobFactory);
+    function ValidationJob(options, context, taskId) {
+        ValidationJob.super_.call(this, logger, options, context, taskId);
+        assert.string(this.context.target);
+        this.nodeId = this.context.target;
+        assert.ok(options.users || this.context.users);
+        this.timeout = options.timeout || 10000;
+        this.retries = options.retries || 2;
+        this.backoffDelay = options.backoffDelay || 1000;
```

Comment:
How about user set these variables to `0`, could you please confirm whether `0` is a valid input, if so, your code need take special handling for number `0`.